src/libs/state.ts

```diff
@@ -92,6 +92,7 @@ export type AuthState = {
   sessionStartTime: number;
   termsOfService: TermsOfServiceStatus;
   terraUser: TerraUser;
+  terraUserAttributes: SamUserAttributes;
 };
```

Comment:
@nawatts have been talking about this, but in general, I think we want to try and consolidate alot of these user related properties like `terraUserProfile`, `terraUser`, and now `terraUserAttributes`.

It may make sense for the future to make TerraUser into a type which contains a TerraUserProfile, a SamUserAttribute, and perhaps the OidcUserProfile.